www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+
+That's right, 20ms. 
+
+This incredible speed is possible because Snowpack only builds files during development as they are requested by the browser. On your very first page load, Snowpack will build each requested file and then cache it for future use. Because each file is built individually, that cached file can be reused forever without ever needing a second build (until you change the file yourself, that is).
+
+`snowpack dev` includes a dev server for hosting, a fully customizable build pipeline for building, and a bunch more familiar features right out of the box:
+
+- TypeScript Support
+- JSX Support
+- Hot Module Replacement (HMR)
+- Importing CSS
+- Importing CSS Modules
+- Importing Images & Other Assets
+- Custom Routing
+- Proxying Requests
+
+**Build Scripts** are 1-line integrations to connect your favorite build tools into your Snowpack build. Thanks to this new idea of O(1) build tooling, every file build can be expressed as a deterministic `input -> build -> output` workflow, which makes Snowpack perfect for Linux-style piping through an already-existing CLI. 
+
+
+```js
+// snowpack.config.json
+{
+  "scripts": {
+    // Pipe every "*.css" file through the PostCSS CLI
+    // stdin (source file) > postcss > stdout (build output)
+    "build:css": "postcss",
+  }
+}
+```
+
+If you've ever used your `package.json` "scripts" config, Snowpack's build scripts should feel familiar by design. We love how `package.json` scripts are able to use your tooling CLIs directly without forcing you to adopt an unnecessary plugin system, and we tried to recreate that magic feeling here.
+
+**Build Plugins** are even more powerful JavaScript integrations for when a CLI just won't cut it. Maybe your favorite build tool doesn't ship a CLI, or that tool's CLI is too slow, or you just want to write your own integration. Build plugins gives you a clean JavaScript interface to extend your build with.
+
+
+```js
+// snowpack.config.json
+{
+  // Connect Babel to your build to run every JS/JSX/TS/TSX file through it
+  "plugins": ["@snowpack/plugin-babel"]
+}
+```
+
+[Check out our full collection of official Snowpack plugins](https://github.com/pikapkg/create-snowpack-app/tree/master/packages), and consider writing your own.
+
+
+## `build` - Optimized, Bundled Builds
+
+![build output example](/img/snowpack-build-example.png)
+
+Run `snowpack build` to build your site for production. This command runs your entire application through your build, outputting a static `build/` directory as a result.
+
+This is where you can now decide to bring in a bundler, if you like. Bundling can result in better compression & networking performance in production, in addition to the minification and optimization steps that Snowpack normally would skip during development. By default, Snowpack's build isn't bundled, but with a single plugin you can integrate Webpack, Parcel, or the bundler of your choice as the final step in your build.
```

Comment:
"Outside of dev, there's a strong case for the benefits of bundling (minification, compression, network optimization). If you'd like to bring in a bundler for your production builds, this is the place.

By default, Snowpack's build isn't bundled. To enable bundling, add the plugin for the bundler of your choice..."

I kind hate how much I used "bundle" in that so pls do a pass of your own and remove as many redundancies as possible